### PR TITLE
Openjdk importer branch

### PR DESCRIPTION
**Pull request by harmonicfunc**
for issue: #1496 
Added a OpenJDK advisory importer that scrapes the vulnerability data from the link: https://openjdk.org/groups/vulnerability/advisories/ and then added the importer to importer registery

### CODE REVIEW

**Comment by harmonicfunc** [comment]
@TG1999 @ambuj-1211 @ziadhany  please review this pr

**Review by keshav-space — CHANGES_REQUESTED** [review]
Thanks @harmonicfunc, use the `VulnerableCodeBaseImporterPipeline` for new importers. See the detailed instructions below.

File: vulnerabilities/importers/openjdk.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+from vulnerabilities import severity_systems
+from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import AffectedPackage
+from vulnerabilities.importer import Importer
+from vulnerabilities.importer import Reference
+from vulnerabilities.importer import VulnerabilitySeverity
+
+
+class OpenJDKImporter(Importer):
```
[the remark this answers is not in the document]
Comment (pombredanne) [reply vulnerabilities/importers/openjdk.py:17]: @keshav-space this would be a nice addition/update to the doc ;)

**Comment by harmonicfunc** [comment]
ok, on it

File: vulnerabilities/importers/openjdk.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (keshav-space) [reply vulnerabilities/importers/openjdk.py:17]: @pombredanne ack, will update the doc here https://vulnerablecode.readthedocs.io/en/latest/contributing.html#writing-an-importer

**Comment by harmonicfunc** [comment]
@keshav-space sorry but do i need to revamp or change logic parts of my code too??

**Comment by pombredanne** [comment]
@harmonicfunc the core logic does not need change, just the integration in a pipeline!
I see you may have some issues with merging ... this is easy at first! You likely want to rebase or restart on the latest main branch and push a new PR, which I would guess is why you closed this?

**Pull request closed without merging**